test/extended/testdata/apibinding/role_hack.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: bind-apiexport-spi
+rules:
+- apiGroups:
+  - apis.kcp.dev
+  resourceNames:
+  - today-cowboys
+  resources:
+  - apiexports
+  verbs:
+  - bind
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: bind-apiexport-spi
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: bind-apiexport-spi
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: "$(USERNAME)"
```

Comment:
@zxiao-redhat any reason you are changing this for a particular user ? we could may be do it for all ?

Reply:
According to my research, ClusterRoleBinding only applies in a specific context under some namespace or user. Therefore, it is not possible to grant permission to all users.

Reply:
@zimo-xiao can you try and see if below works ? I have been following this way
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: system:kcp:authenticated:apiexport:tenancy:bind
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: system:kcp:apiexport:tenancy:bind
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: system:authenticated
```

Reply:
Hi @kasturinarra, thank you for the suggestion. I had just used you method and updated the repo. PTAL. Thanks!
Test: passed: (45.1s) 2023-02-09T10:45:38 "[area/apiexports] Author:zxiao-Critical-[KCP] Verify if APIBinding binds with exported custom resource [Suite:kcp/smoke/parallel]"

Reply:
@zimo-xiao also do not see the updated fies or am i missing something ? could you please help check ? thanks !!

Reply:
Hi @kasturinarra I apologize. 😂 I committed my changes to another branch. I just updated my code base, please take a look.

Test: passed: (26.6s) 2023-02-12T14:05:14 "[area/apiexports] Author:zxiao-Critical-[KCP] Verify if APIBinding binds with exported custom resource [Suite:kcp/smoke/parallel]"